inc/functions/cloudflare.php

```diff
@@ -740,21 +740,5 @@ function rocket_auto_purge_cloudflare_by_url( $post, $purge_urls, $lang ) {
 
 	// Purge CloudFlare.
 	$cf_purge = rocket_purge_cloudflare_by_url( $post, $purge_urls, $lang );
```

Comment:
No need to assign the result to `$cf_purge` then.
```suggestion
	rocket_purge_cloudflare_by_url( $post, $purge_urls, $lang );
```